components/Layout/index.js

```diff
@@ -0,0 +1,13 @@
+import NavigationBar from "../NavigationBar";
+import Header from "../Header";
+import Wrapper from "../Wrapper";
+
+export default function Layout() {
+  return (
+    <>
+      <Header></Header>
+      <Wrapper></Wrapper>
+      <NavigationBar></NavigationBar>
+    </>
+  );
+}
```

Comment:
```suggestion
export default function Layout({children}) {
  return (
    <>
      <Header />
      <Wrapper>{children}</Wrapper>
      <NavigationBar/ >
    </>
  );
}
```
This is probably better. If you're defining the content of `Header` and `Navigation` inside the components, you can use self-closing JSX elements.